integration-tests/src/test/groovy/com/okta/sdk/tests/it/PolicyRulesIT.groovy

```diff
@@ -0,0 +1,298 @@
+                    .setAccess(OktaSignOnPolicyRuleSignonActions.AccessEnum.ALLOW)
+                    .setRequireFactor(true)
+                    .setFactorPromptMode(OktaSignOnPolicyRuleSignonActions.FactorPromptModeEnum.ALWAYS)
+                    .setRememberDeviceByDefault(false)
+                    .setSession(client.instantiate(OktaSignOnPolicyRuleSignonSessionActions)
+                        .setUsePersistentCookie(false)
+                        .setMaxSessionIdleMinutes(720)
+                        .setMaxSessionLifetimeMinutes(0))))
+            .setConditions(client.instantiate(OktaSignOnPolicyRuleConditions)
+                .setNetwork(client.instantiate(PolicyNetworkCondition)
+                    .setConnection(PolicyNetworkCondition.ConnectionEnum.ANYWHERE))
+                .setAuthContext(client.instantiate(PolicyRuleAuthContextCondition)
+                    .setAuthType(PolicyRuleAuthContextCondition.AuthTypeEnum.RADIUS)))
+            .setName(policyRuleName))
+        registerForCleanup(policyRule)
+
+        assertThat policyRule.getConditions().getAuthContext().getAuthType(), is(PolicyRuleAuthContextCondition.AuthTypeEnum.RADIUS)
+        assertThat policyRule.getConditions().getNetwork().getConnection(), is(PolicyNetworkCondition.ConnectionEnum.ANYWHERE)
+    }
+
+    @Test
+    def createOktaSignOnCloudPolicyRule() {
+
+        def policy = client.createPolicy(client.instantiate(OktaSignOnPolicy)
+            .setName("policy+" + UUID.randomUUID().toString())
+            .setDescription("IT created Policy - CRUD rules"))
+        registerForCleanup(policy)
+
+        def policyRuleName = "policyRule+" + UUID.randomUUID().toString()
+        OktaSignOnPolicyRule policyRule = policy.createRule(client.instantiate(OktaSignOnPolicyRule)
+            .setActions(client.instantiate(OktaSignOnPolicyRuleActions)
+                .setSignon(client.instantiate(OktaSignOnPolicyRuleSignonActions)
+                    .setAccess(OktaSignOnPolicyRuleSignonActions.AccessEnum.ALLOW)
+                    .setRequireFactor(true)
+                    .setFactorPromptMode(OktaSignOnPolicyRuleSignonActions.FactorPromptModeEnum.ALWAYS)
+                    .setRememberDeviceByDefault(false)
+                    .setSession(client.instantiate(OktaSignOnPolicyRuleSignonSessionActions)
+                        .setUsePersistentCookie(false)
+                        .setMaxSessionIdleMinutes(720)
+                        .setMaxSessionLifetimeMinutes(0))))
+            .setConditions(client.instantiate(OktaSignOnPolicyRuleConditions)
+                .setNetwork(client.instantiate(PolicyNetworkCondition)
+                    .setConnection(PolicyNetworkCondition.ConnectionEnum.ANYWHERE))
+                .setAuthContext(client.instantiate(PolicyRuleAuthContextCondition)
+                    .setAuthType(PolicyRuleAuthContextCondition.AuthTypeEnum.RADIUS))
+                .setPeople(client.instantiate(PolicyPeopleCondition)
```

Comment:
AuthType should be `ANY` for this scenario.

Reply:
🤔 i would have expected this test to fail

Reply:
Doh, the test has a return type, so this test wasn't being run.  Looks like it was the only on though.
Good catch!